<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSerialDialog</class>
 <widget class="QDialog" name="ConfigSerialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confguration: serial port</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1,0">
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="grbDataSize">
     <property name="title">
      <string>Data size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QRadioButton" name="rbDataSize8bits">
        <property name="text">
         <string>8 bits</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupDataSize</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDataSize7bits">
        <property name="text">
         <string>7 bits</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupDataSize</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDataSize6bits">
        <property name="text">
         <string>6 bits</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupDataSize</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDataSize5bits">
        <property name="text">
         <string>5 bits</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupDataSize</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QGroupBox" name="grbParity">
     <property name="title">
      <string>Parity</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QRadioButton" name="rbParityNone">
        <property name="text">
         <string>None</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupParity</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbParityOdd">
        <property name="text">
         <string>Odd</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupParity</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbParityEven">
        <property name="text">
         <string>Even</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupParity</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbParityMark">
        <property name="text">
         <string>Mark</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupParity</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbParitySpace">
        <property name="text">
         <string>Space</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupParity</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="grbStopBits">
     <property name="title">
      <string>Stop bits</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QRadioButton" name="rbStopBits1">
        <property name="text">
         <string>1 bit</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupStopBits</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStopBits1p5">
        <property name="text">
         <string>1.5 bit</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupStopBits</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStopBits2">
        <property name="text">
         <string>2 bits</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupStopBits</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTableWidget" name="tableParams">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Param</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <widget class="QGroupBox" name="grbFlowControl">
     <property name="title">
      <string>Flow control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QRadioButton" name="rbFlowCtlNone">
        <property name="text">
         <string>None</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupFlowControl</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbFlowCtlSoft">
        <property name="text">
         <string>Software</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupFlowControl</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbFlowCtlHard">
        <property name="text">
         <string>Hardware</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">rbgroupFlowControl</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="grbPort">
     <property name="title">
      <string>Port</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="lblBaudrate">
        <property name="text">
         <string>Baudrate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbBaudrate">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbName">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnHelp">
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rbgroupStopBits"/>
  <buttongroup name="rbgroupParity"/>
  <buttongroup name="rbgroupFlowControl"/>
  <buttongroup name="rbgroupDataSize"/>
 </buttongroups>
</ui>
